### ISSUE TITLE
BUGS file removal breaks build

### ISSUE THREAD

**Issue by The-Compiler** [herbstluftwm/herbstluftwm#1093]
1282c4f976f70e68d5878d67a32c5faac19009d5 breaks cmake:

```
Install the project...
-- Install configuration: "Release"
CMake Error at cmake_install.cmake:46 (file):
  file INSTALL cannot find
  "/home/florian/.cache/yay/herbstluftwm-git/src/herbstluftwm/BUGS": No such
  file or directory.
```

Also the file is still referenced in various docs as well:

````console
$ git grep BUGS                                              
CMakeLists.txt:install(FILES BUGS NEWS DESTINATION ${DOCDIR})
INSTALL:A list of known bugs is listed in BUGS. If you found other bugs or want to
NEWS:things like EWMH or rules aren't implemented yet (See BUGS file for a list of
doc/herbstclient.txt:BUGS
doc/herbstclient.txt:See the *herbstluftwm* distribution BUGS file.
doc/herbstluftwm.txt:BUGS
doc/herbstluftwm.txt:See the *herbstluftwm* distribution BUGS file.
src/client.cpp:         * twice.  see BUGS for more information
```

**Comment by t-wissmann** [comment]
Sorry about this!

**Comment by t-wissmann** [comment]
@blarz fixed it in #1095. :-)
